.github/workflows/ci.yml

```diff
@@ -45,7 +45,6 @@ jobs:
           - "markduplicates"
           - "mpileup"
           - "msisensorpro"
-          - "mutect2"
```

Comment:
we still want to have the mutect tests that pass though, I would say

Reply:
if you add the functioning mutect tests back in, they will be triggered. And yes the manual one not, because you changed the tag to mutect_manuel :)